packages/angular_devkit/build_angular/src/browser/index2.ts

```diff
@@ -206,6 +249,72 @@ export type BrowserConfigTransformFn = (
   config: webpack.Configuration,
 ) => Observable<webpack.Configuration>;
 
+function mergeBuildResult(buildEvents: BuildResult[]): BuildResult {
+
+  if (buildEvents.length === 1) {
+    return buildEvents[0];
+  }
+
+  const [emit1, emit2] = buildEvents;
+
+  return {
+    emittedFiles: [...emit1.emittedFiles, ...emit2.emittedFiles],
+    error: buildEvents[0].error + '\n' + buildEvents[0].error,
+    info: { ...buildEvents[0].info, ...buildEvents[1].info },
+    success: buildEvents[0].success && buildEvents[1].success,
+    target: buildEvents[0].target,
+  };
+}
+
+async function writeIndexHtml(
+  buildEvents: BuildResult[],
+  options: BrowserBuilderSchema,
+  root: Path,
+  host: virtualFs.Host<fs.Stats>,
+) {
+
+  const indexBaseName = normalize(basename(normalize(options.index)));
+  const indexFileName = join(root, normalize(options.index));
+  const outputFileName = join(root, normalize(options.outputPath), indexBaseName);
+  const indexBuffer = await host.read(indexFileName).toPromise();
+  const indexContent = virtualFs.fileBufferToString(indexBuffer);
+
+  const emit1 = buildEvents[0].emittedFiles || [];
+  const emit2 = buildEvents[1].emittedFiles || [];
+
+  // TODO: We need to sort those files according to its entrypoints
+
+  const files1: CompiledFileInfo[] = emit1.map(f => ({
+    file: f.file,
+    type: f.extension === 'js' ? 'nomodule' : 'none',
+  }));
+
+  const files2: CompiledFileInfo[] = emit2.map(f => ({
+    file: f.file,
+    type: f.extension === 'js' ? 'module' : 'none',
+  }));
+
+  const source = await generateIndexHtml({
+    input: options.index,
+    inputContent: indexContent,
+    baseHref: options.baseHref,
+    sri: options.subresourceIntegrity || false,
+    loadOutputFile: async (fileName: string) => {
+      const path = join(root, normalize(fileName), normalize(fileName));
+      const buffer = await host.read(path).toPromise();
+
+      return virtualFs.fileBufferToString(buffer);
+    },
+    unfilteredSortedFiles: [
+      ...files1,
+      ...files2,
+    ],
+    noModuleFiles: new Set<string>(['polyfills.es5-es5.js']),
```

Comment:
As per slack convo `polyfills.es5.js `

Reply:
I think it's best, to do this with another PR to respect @alexeagle's feedback here: https://github.com/angular/angular-cli/pull/14061#issuecomment-479990716)